packages/idos-sdk-js/package.json

```
@@ -10,13 +10,13 @@
   "files": ["dist"],
   "exports": {
     ".": {
-      "import": "./dist/idos-sdk.js",
-      "types": "./dist/idos-sdk.d.ts"
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
     }
   },
   "scripts": {
-    "dev": "vite build --mode development --watch",
-    "build": "vite build",
+    "dev": "dotenvx run --convention=nextjs -- tsup --watch",
+    "build": "export NODE_ENV=production && dotenvx run --convention=nextjs -- tsup",
```

Maybe we want to do this instead?
```suggestion
    "build": "dotenvx run --file=.env.production -- tsup",
```

If this doesn't look good at first try, let's just keep what you have (since it'll never surprise anyone).